cumulusci.yml

```diff
@@ -399,6 +427,24 @@ flows:
                     directories: unpackaged/config/wave
                     really_change_files: True
 
+    add_nmg_permissions:
+        descriptions: Extracts newtwork and permissions then generates a new files to load permissions for network memeber group
+        steps:
+            1:
+                task: extract_dataset
+                options: 
+                    mapping: datasets/mapping-nmg-extract.yml
+                    sql_path: datasets/nmg-extract.sql
+            2:
+                task: add_permissions_to_nmg
+            3:
+                task: load_dataset
+                options:
+                    mapping: datasets/mapping-nmg-load.yml
+                    sql_path: datasets/nmg-load.sql
+            4:
+                task: reset_nmg_sql
```

Comment:
Something seems a bit weird about modifying a file and then resetting it. Can't you just make a brand new file and then delete or ignore it?

Why does the repository need to contain datasets/nmg-load.sql at all, if it is just a copy of the template? Instead of appending to it couldn't you read the template, write it, and then continue adding to it? I might miss something.

Reply:
This way feels a bit risky because if the flow fails (which they often do) then the file could be in a weird state and you'd append to a file in the wrong state.